public/app/models/ListPager.js

```diff
@@ -0,0 +1,76 @@
+(function () {
+    'use strict';
+
+    define(
+        [
+            'lodash',
+            'knockout'
+        ],
+        function (_, ko) {
```

Comment:
This module takes a big list and presents a smaller list that represents a page at a time of the big list.